Gems/Multiplayer/Assets/TranslationAssets/AZEvents/OnResetCountChangedEvent.names

```diff
@@ -0,0 +1,56 @@
+{
+    "entries": [
+        {
+            "base": "On ResetCount Changed Event",
+            "context": "AZEventHandler",
+            "variant": "",
+            "details": {
+                "name": "On Reset Count Changed Event"
+            },
+            "slots": [
+                {
+                    "base": "New uint8_t",
+                    "details": {
+                        "name": "New uint8_t"
```

Comment:
This seems that it should be called something other than "New uint8_t"

Reply:
I gave it a proper name. Done.